lib/meta.nix

```diff
@@ -484,4 +489,226 @@ rec {
     assert assertMsg (match ".*/.*" y == null)
       "lib.meta.getExe': The second argument \"${y}\" is a nested path with a \"/\" character, but it should just be the name of the executable instead.";
     "${getBin x}/bin/${y}";
+
+  /**
+    `tryEval` version of [`lib.meta.cpeFullVersionWithVendor`](#function-library-lib.meta.cpeFullVersionWithVendor). Always returns `{success = true; value = cpeFullVersionWithVendor vendor version;}`, because cpeFullVersionWithVendor always succeeds.
```

Comment:
I don't think we need such a function at all then. The only use case of it is the internal code, which can just be refactored to call `cpeFullVersionWithVendor` instead.